tcpdf.php

```diff
@@ -1832,6 +1832,14 @@ class TCPDF {
 	 */
 	protected $gdgammacache = array();
 
+	/**
+	 * Whether to allow local file path in image html tags, when prefixed with file://
+     *
+	 * @protected
+	 * @since 6.4 (2020-07-23)
```

Comment:
I assumed adding this new feature would require a 6.3.0 release. feel free to tell me what you want me to put in here.